src/core/context.ts

```diff
@@ -93,6 +93,7 @@ export function createRoutesContext(options: ResolvedOptions) {
   }
 
   async function addPage({ filePath: path, routePath }: HandlerContext) {
+    path = slash(path)
```

Comment:
let's do this once inside the RoutesFolderWatcher class instead

Reply:
This will make the code more elegant, thank you, I have updated